infra/cifuzz/continuous_integration.py

```diff
@@ -132,11 +133,12 @@ def prepare_for_fuzzer_build(self):
     assert self.config.pr_ref or self.config.commit_sha
     # detect_main_repo builds the image as a side effect.
     inferred_url, image_repo_path = (build_specified_commit.detect_main_repo(
-        self.config.project_name, repo_name=self.config.project_repo_name))
+        self.config.oss_fuzz_project_name,
+        repo_name=self.config.project_repo_name))
 
     if not inferred_url or not image_repo_path:
       logging.error('Could not detect repo from project %s.',
```

Comment:
I have similar thoughts on this log message as well. We probably don't need to include the project name in this log message.